requires/testing.txt

```diff
@@ -1,5 +1,14 @@
 coverage>3,<5
 codecov
-mock
+flake8
+flake8-comprehensions
+flake8-deprecated
+flake8-html
+flake8-import-order
+flake8-quotes
+flake8-rst-docstrings
+flake8-tuple
 nose>=1.3.1,<2.0.0
+sprockets.mixins.mediatype>=3.0.0
+u-msgpack-python
```

Comment:
Tests explicitly reference this package so this is fine. However, *at some point* the tests should be switched to use sprockets.mixins.mediatype directly.